PowerPointLabs/PowerPointLabs/ShortcutsLab/ConvertToPicture.cs

```diff
@@ -34,22 +35,17 @@ public static void Convert(PowerPointPresentation pres, PowerPointSlide slide, P
             }
         }
 
-        public static void ConvertAndSave(PowerPoint.Selection selection, string fileName)
+        public static bool ConvertAndSave(ShapeRange selectedShapes, string fileName)
         {
-            if (ShapeUtil.IsSelectionShapeOrText(selection))
+            try
             {
-                if (selection.HasChildShapeRange)
-                {
-                    GraphicsUtil.ExportShape(selection.ChildShapeRange, fileName);
-                }
-                else
-                {
-                    GraphicsUtil.ExportShape(selection.ShapeRange, fileName);
-                }
+                GraphicsUtil.ExportShape(selectedShapes, fileName);
+                return true;
             }
-            else
+            catch
             {
                 MessageBox.Show(ShortcutsLabText.ErrorTypeNotSupported, ShortcutsLabText.ErrorWindowTitle);
```

Comment:
You might want to only show this error "Type is not supported" if` ShapeUtil.IsSelectionShapeOrText(selectedShapes) `is False. Thus if it goes to the `catch` portion, show a more generic error message instead.